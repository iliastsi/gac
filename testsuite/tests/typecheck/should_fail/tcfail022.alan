foo(x : int) : int
{
  return 42;
}
