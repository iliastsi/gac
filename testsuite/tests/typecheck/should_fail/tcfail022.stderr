tcfail022.alan:1:1-18: Error: Type mismatch at `foo(x :  int) : int'
	Outermost function has to take no arguments

